common/src/main/java/io/homeassistant/companion/android/database/widget/MediaPlayerControlsWidgetEntity.kt

```diff
@@ -18,6 +18,8 @@ data class MediaPlayerControlsWidgetEntity(
     val showSeek: Boolean,
     @ColumnInfo(name = "showVolume")
     val showVolume: Boolean,
+    @ColumnInfo(name = "showSource", defaultValue = "true")
```

Comment:
Should this default to `true`? It will change the appearance of existing widgets.

Reply:
I can change this to false